ICSharpCode.ILSpyX/LoadedAssembly.cs

```diff
@@ -353,7 +353,7 @@ async Task<LoadResult> LoadAsync(Task<Stream?>? streamTask)
 				}
 			}
 
-			if (result?.IsSuccess != true)
+			if (result?.IsSuccess != true || result.Package?.Kind == LoadedPackage.PackageKind.Zip) // workaround to check this before zip
```

Comment:
we cannot do this, because LoadedAssembly should not need to know about any format other than PE.